cvat-ui/src/components/header/settings-modal/settings-modal.tsx

```diff
@@ -29,9 +72,16 @@ const SettingsModal = (props: SettingsModalProps): JSX.Element => {
             width={800}
             className='cvat-settings-modal'
             footer={(
-                <Button type='primary' onClick={onClose}>
-                    Close
-                </Button>
+                <>
+                    <Tooltip title='Will save settings from this page and appearance settings on standard workspace page in browser'>
+                        <Button type='primary' onClick={onSaveSettings}>
+                            Save
+                        </Button>
+                    </Tooltip>
+                    <Button type='default' onClick={onClose}>
+                        Close
+                    </Button>
```

Comment:
Need to update dock. Do we need button close? We have "X" and also click everywhere out of  the modal closes the modal

Reply:
It's default behaviour for the modal element.